lib/src/mobile_scanner_controller.dart

```diff
@@ -82,6 +83,11 @@ class MobileScannerController extends ValueNotifier<MobileScannerState> {
   /// Defaults to false, and is only supported on iOS, MacOS and Android.
   final bool returnImage;
 
+  /// Whether the image should be inverted in intervals (original - inverted - original…)
```

Comment:
```suggestion
  /// Whether the scanner should try to detect color-inverted barcodes in every other frame.
  ///
  /// When this option is enabled, every odd frame from the camera preview has its colors inverted before processing.
  /// This is useful if barcodes can be both black-on-white (the most common) and white-on-black (less common).
  /// Usage of this parameter can incur a performance cost, as some frames need to be altered further during processing.
  ///
  /// Defaults to false and is only supported on Android and iOS.
```

We should provide users with a detailed description of what this does.